pubspec.lock

```diff
@@ -650,26 +650,26 @@ packages:
     dependency: transitive
     description:
       name: leak_tracker
-      sha256: "6bb818ecbdffe216e81182c2f0714a2e62b593f4a4f13098713ff1685dfb6ab0"
+      sha256: "8dcda04c3fc16c14f48a7bb586d4be1f0d1572731b6d81d51772ef47c02081e0"
```

Comment:
This diff doesn't seem related to what this commit is doing. To keep the commits [clear and coherent](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html), this change should be kept to a separate commit.

It should also go with a corresponding change in `pubspec.yaml`. See our README about upgrading Flutter.